data-rights-protocol.md

```diff
@@ -0,0 +1,232 @@
+
+| name                                      | description                                                                                                                                                                                                 |
+|-------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `aud`                                     | audience claim MUST contain to the legal entity which is responsible for processing rights action                                                                                                           |
+| `iss`                                     | issuer claim MUST contain the legal entity which is making the request on behalf of user                                                                                                                    |
+| `sub`                                     | if known, subject claim SHALL contain the Covered Business's preferred public identifier for the user.                                                                                                      |
+| `name`                                    | if known, claim SHALL contain the user's full name most likely known by the Covered Business                                                                                                                |
+| `email` or `email_verified`               | if known, claim SHALL contain the user's email address, if known. `email_verified` MUST only contain a value if this address was verified by the agent                                                      |
+| `phone_number` of `phone_number_verified` | if known, claim SHALL contain the user's phone number, if known. `phone_number_verified` MUST only contain a value if this address was verified by the agent through a phone call or SMS one-time password. |
+| `address`                                 | if known, claim SHALL contain the user's preferred address, if known.                                                                                                                                       |
+
+Covered Businesses SHALL make the final determination over whether a given token is trustworthy. Agents SHALL make a best effort to provide trustworthy tokens, by attaching user-attested claims as available, and by ensuring their JWTs are signed by a key which the Covered Businesses and PIPs can verify against. 
+
+JWTs should probably not be encrypted? managing the encryption key exchange here is very strange and necessarily happening out-of-scope of the protocol. but we already have shared-secret API authentication in section 3.07; I am queasy about having user tokens identity floating in the open here....
+
+### 3.06 Error States
+
+Todo, error states in processing, error states in POST exercise, etc...
```

Comment:
still punting on thinking about this, really need to spend some time considering it more before unveiling.